integration-tests/test-cases/scenario-tests/src/test/java/io/cellery/integration/scenario/tests/cli/ImageBasedTestCase.java

```diff
@@ -29,40 +29,52 @@
  * This test includes the test cases related to the CLI and images.
  */
 public class ImageBasedTestCase extends BaseTestCase {
-    private final String orgName = "wso2cellery";
-    private final String imageName = "hello-world-cell";
-    private final String employeeImage = "employee-test-cell";
-    private final String employeeVersion = "latest";
+    private static final String helloWorldImage = "hello-world-cell";
+    private static final String employeeImage = "employee-test-cell";
 
     @Test
+    public void buildResourceImage() throws Exception {
```

Comment:
Lets move this logic to a @BeforeClass method, and get rid of the dependsOnMethods annotations.